gravitee-apim-console-webui/src/components/gio-side-nav/gio-side-nav.component.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { InteractivityChecker } from '@angular/cdk/a11y';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { GioLicenseExpirationNotificationHarness, LICENSE_CONFIGURATION_TESTING } from '@gravitee/ui-particles-angular';
+import { ActivatedRoute } from '@angular/router';
+import { of } from 'rxjs';
+
+import { GioSideNavHarness } from './gio-side-nav.harness';
+import { GioSideNavComponent } from './gio-side-nav.component';
+import { GioSideNavModule } from './gio-side-nav.module';
+
+import { License } from '../../entities/license/License';
+import { GioPermissionService } from '../../shared/components/gio-permission/gio-permission.service';
+import { CONSTANTS_TESTING, GioHttpTestingModule } from '../../shared/testing';
+describe('GioSideNavComponent', () => {
+  let fixture: ComponentFixture<GioSideNavComponent>;
+  let httpTestingController: HttpTestingController;
+  let loader: HarnessLoader;
+
+  const expirationDateInOneYear = new Date();
+  expirationDateInOneYear.setFullYear(expirationDateInOneYear.getFullYear() + 1);
+
+  const init = async (licenseNotificationEnabled = true, hasLicenseMgmtPermission = true) => {
+    await TestBed.configureTestingModule({
+      declarations: [GioSideNavComponent],
+      imports: [NoopAnimationsModule, GioHttpTestingModule, GioSideNavModule, MatIconTestingModule],
+      providers: [
+        {
+          provide: GioPermissionService,
+          useValue: {
+            hasAnyMatching: (permissions: string[]) => {
+              return !(!permissions.includes('organization-license_management-r') || !hasLicenseMgmtPermission);
+            },
+          },
+        },
+        {
+          provide: 'Constants',
+          useFactory: () => {
+            const constants = CONSTANTS_TESTING;
+            constants.org.settings = { ...constants.org.settings, licenseExpirationNotification: { enabled: licenseNotificationEnabled } };
+            constants.org.environments = [{ id: 'DEFAULT', name: 'default', hrids: [], organizationId: 'organizationId' }];
+
+            return constants;
+          },
+        },
+        { provide: 'LicenseConfiguration', useValue: LICENSE_CONFIGURATION_TESTING },
+        { provide: ActivatedRoute, useValue: { params: of({ envId: 'DEFAULT' }) } },
+      ],
+    })
+      .overrideProvider(InteractivityChecker, {
+        useValue: {
+          isFocusable: () => true, // This traps focus checks and so avoid warnings when dealing with
+          isTabbable: () => true, // This traps focus checks and so avoid warnings when dealing with
+        },
+      })
+      .compileComponents();
+
+    fixture = TestBed.createComponent(GioSideNavComponent);
+    httpTestingController = TestBed.inject(HttpTestingController);
+    loader = TestbedHarnessEnvironment.loader(fixture);
```

Comment:
Just a hint.
It seems to me that it is possible to use  : `harnessForFixture` to avoid `harness = await loader.getHarnessOrNull(GioSideNavHarness);` on each test.

Reply:
I'm deleting the `GioSideNavHarness` since it doesn't really do anything.